middleware/fastcgi/fastcgi.go

```diff
@@ -34,12 +36,23 @@ type Handler struct {
 	Rules []Rule
 }
 
+func (h Handler) DoesFileExist(path string) bool {
+	file := h.Root + path
+	if _, err := os.Stat(file); err == nil {
+		return true
+	}
+	return false
+}
+
 // ServeHTTP satisfies the middleware.Handler interface.
 func (h Handler) ServeHTTP(w http.ResponseWriter, r *http.Request) (int, error) {
 	servedFcgi := false
+	indexFile := "index.php"
```

Comment:
Yep - we sat down yesterday and talked about this. I'll be working on it today to generalize it. Thanks for double-checking @norwack! We're both glad to have the feedback.